nltk/tokenize/stanford_segmenter.py

```diff
@@ -82,7 +83,8 @@ def __init__(self,
         self._encoding = encoding
         self.java_options = java_options
         options = {} if options is None else options
-        self._options_cmd = ','.join('{0}={1}'.format(key, json.dumps(val)) for key, val in options.items())
+        self._options_cmd = ','.join('{0}={1}'.format(
```

Comment:
Oh, I have to confess I dunno if there's a linter check as part of the CI tests, but personally I am really not that big on enforcing the line length limit. In this particular case, you could split the line right before `for key, val ...` and I think you might still be within 80 characters ;)